app/models/ems_refresh/save_inventory_middleware.rb

```diff
@@ -41,9 +41,7 @@ def save_middleware_server_groups_inventory(domain, hashes, target = nil)
     hashes.each do |h|
       h[:domain_id] = domain[:id]
     end
-
-    save_inventory_multi(domain.middleware_server_groups, hashes, deletes, [:ems_ref], nil,
-                         [:middleware_domain])
+    save_inventory_multi(domain.middleware_server_groups, hashes, deletes, [:ems_ref], nil, [:middleware_domain])
```

Comment:
oh right, it's only a "whitespace change" (there was no reason for the line break)